Description: Update doctests to GLPK 4.60
 Taken from https://trac.sagemath.org/ticket/20710
Author: Jeroen Demeyer <jdemeyer@cage.ugent.be>
Bug: https://trac.sagemath.org/ticket/20710
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/src/sage/numerical/backends/glpk_backend.pyx b/src/sage/numerical/backends/glpk_backend.pyx
index 7a7ae0a..b0f9f2a 100644
--- a/sage/src/sage/numerical/backends/glpk_backend.pyx
+++ b/sage/src/sage/numerical/backends/glpk_backend.pyx
@@ -1004,8 +1006,8 @@ cdef class GLPKBackend(GenericBackend):
         Same, now with a time limit::
 
             sage: p.solver_parameter("mip_gap_tolerance",1)
-            sage: p.solver_parameter("timelimit",0.01)
-            sage: p.solve() # rel tol 1
+            sage: p.solver_parameter("timelimit",3.0)
+            sage: p.solve() # rel tol 100
             1
         """
 
diff --git a/src/sage/numerical/backends/glpk_graph_backend.pyx b/src/sage/numerical/backends/glpk_graph_backend.pyx
index a099788..498e8e1 100644
--- a/sage/src/sage/numerical/backends/glpk_graph_backend.pyx
+++ b/sage/src/sage/numerical/backends/glpk_graph_backend.pyx
@@ -162,12 +162,12 @@ cdef class GLPKGraphBackend(object):
         sage: a = gbe.add_edge('0', '1')
         sage: gbe.write_graph(SAGE_TMP+"/graph.txt")
         Writing graph to ...
-        2 lines were written
+        4 lines were written
         0
         sage: gbe1 = GLPKGraphBackend(SAGE_TMP+"/graph.txt", "plain")
         Reading graph from ...
-        Graph has 2 vertices and 1 arc
-        2 lines were read
+        Graph has 2 vertices and 1 edge
+        3 lines were read
 
     The following example imports a Sage ``Graph`` and then uses it to solve a
     maxflow problem::
@@ -1060,7 +1060,7 @@ cdef class GLPKGraphBackend(object):
             sage: a = gbe.add_edge("0", "1")
             sage: gbe.write_graph(SAGE_TMP+"/graph.txt")
             Writing graph to ...
-            2 lines were written
+            4 lines were written
             0
         """
 
diff --git a/src/sage/numerical/mip.pyx b/src/sage/numerical/mip.pyx
index 8c17d87..b45c82f 100644
--- a/sage/src/sage/numerical/mip.pyx
+++ b/sage/src/sage/numerical/mip.pyx
@@ -2519,10 +2519,10 @@ cdef class MixedIntegerLinearProgram(SageObject):
             sage: b.solver_parameter("simplex_or_intopt", "simplex_only")
             sage: b.solver_parameter("verbosity_simplex", "GLP_MSG_ALL")
             sage: p.solve()  # rel tol 1e-5
-            GLPK Simplex Optimizer, v4.55
+            GLPK Simplex Optimizer, v4.60
             2 rows, 2 columns, 4 non-zeros
-            *     0: obj =   7.000000000e+00  infeas =  0.000e+00 (0)
-            *     2: obj =   9.400000000e+00  infeas =  0.000e+00 (0)
+            *     0: obj =   7.000000000e+00 inf =   0.000e+00 (2)
+            *     2: obj =   9.400000000e+00 inf =   0.000e+00 (0)
             OPTIMAL LP SOLUTION FOUND
             9.4
         """
